packages/vgrammar/__tests__/mark/element.test.ts

```diff
@@ -42,6 +42,7 @@ test('Element executes encoding', function () {
     {
       enter: { x: 123 }
     },
+    false,
```

Comment:
maybe you need to add a test case for `reenter`

Reply:
Fix. Added in mark.test.ts file.